patches/000-initial-setup.patch

```diff
@@ -573,14 +699,14 @@ index 69c509a771..05eac35d88 100644
  }
  
  func TestShortSessionKey(t *testing.T) {
-+	if boring.Enabled {
++	if boring.Enabled && ! boringtest.Supports(t, "PKCSv1.5") {
```

Comment:
nit: Do you mind cleaning up the space in the negation operator? E.g. `!assertion()` as opposed to `! assertion()`.